share/spice/uv/uv.handlebars

```diff
@@ -0,0 +1,4 @@
+<span>{{name}}</span>
+<a href="http://some.url/profile/data1">
+   <img src="/iu/?u=Place a image url here">
+</a>
```

Comment:
As you're returning the data using `normalize` you won't need a handlebars template. This can be removed.